activation/handler_test.go

```diff
@@ -144,1069 +208,537 @@ func newTestHandler(tb testing.TB, goldenATXID types.ATXID) *testHandler {
 	}
 }
 
-func TestHandler_processBlockATXs(t *testing.T) {
```

Comment:
:bulb: This test was removed because it was testing if the `GetEpochWeight()` works properly, which has nothing to do with the ATX Handler.